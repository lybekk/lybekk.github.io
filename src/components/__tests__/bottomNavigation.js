import React from "react"
import renderer from "react-test-renderer"

import BottomNavigation from "../bottomNavigation"

describe("BottomNavigation", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(<BottomNavigation />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})