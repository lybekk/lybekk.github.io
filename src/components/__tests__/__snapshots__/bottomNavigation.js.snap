// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomNavigation renders correctly 1`] = `
<div
  className="bottomNavigation attentionShimmer"
>
  <div>
    <a
      href="/"
    >
      <button
        aria-controls="dropdown-menu4"
        aria-haspopup="true"
        className="button is-link is-inverted is-rounded"
      >
        <span
          className="icon is-small"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-home fa-w-18 "
            data-icon="home"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </span>
      </button>
    </a>
  </div>
  <div
    className="dropdown is-hoverable is-up is-right"
  >
    <div
      className="dropdown-trigger"
    >
      <button
        aria-controls="dropdown-menu4"
        aria-haspopup="true"
        className="button is-link is-inverted is-rounded"
      >
        <span
          className="icon"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-paper-plane fa-w-16 "
            data-icon="paper-plane"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M476 3.2L12.5 270.6c-18.1 10.4-15.8 35.6 2.2 43.2L121 358.4l287.3-253.2c5.5-4.9 13.3 2.6 8.6 8.3L176 407v80.5c0 23.6 28.5 32.9 42.5 15.8L282 426l124.6 52.2c14.2 6 30.4-2.9 33-18.2l72-432C515 7.8 493.3-6.8 476 3.2z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </span>
      </button>
    </div>
    <div
      className="dropdown-menu"
      role="menu"
    >
      <div
        className="dropdown-content"
      >
        <div
          className="dropdown-item"
        >
          <p>
            Give feedback
          </p>
        </div>
        <hr
          className="dropdown-divider"
        />
        <div
          className="dropdown-item"
        >
          <a
            className="has-text-link"
            href="mailto:feedback@lybekk.tech?subject=Hi!&body=%0D%0A---%0D%0ARequest%20made%20at%3A%20http://localhost/"
            rel="noreferrer"
            target="_blank"
          >
            <span
              className="icon is-small mr-1"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-envelope fa-w-16 "
                data-icon="envelope"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M502.3 190.8c3.9-3.1 9.7-.2 9.7 4.7V400c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V195.6c0-5 5.7-7.8 9.7-4.7 22.4 17.4 52.1 39.5 154.1 113.6 21.1 15.4 56.7 47.8 92.2 47.6 35.7.3 72-32.8 92.3-47.6 102-74.1 131.6-96.3 154-113.7zM256 320c23.2.4 56.6-29.2 73.4-41.4 132.7-96.3 142.8-104.7 173.4-128.7 5.8-4.5 9.2-11.5 9.2-18.9v-19c0-26.5-21.5-48-48-48H48C21.5 64 0 85.5 0 112v19c0 7.4 3.4 14.3 9.2 18.9 30.6 23.9 40.7 32.4 173.4 128.7 16.8 12.2 50.2 41.8 73.4 41.4z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </span>
            Email
          </a>
        </div>
        <div
          className="dropdown-item"
        >
          <a
            className="has-text-link"
            href="https://docs.google.com/forms/d/e/1FAIpQLScdBp2iPp4vd1NCfwmAbl5NMOzYj4LaRtiPUuGK-rBfXkyUEA/viewform?usp=pp_url&entry.1456738885=None+needed"
            rel="noreferrer"
            target="_blank"
          >
            <span
              className="icon is-small mr-1"
            >
              <svg
                aria-hidden="true"
                className="svg-inline--fa fa-file-alt fa-w-12 "
                data-icon="file-alt"
                data-prefix="fas"
                focusable="false"
                role="img"
                style={Object {}}
                viewBox="0 0 384 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M224 136V0H24C10.7 0 0 10.7 0 24v464c0 13.3 10.7 24 24 24h336c13.3 0 24-10.7 24-24V160H248c-13.2 0-24-10.8-24-24zm64 236c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12v8zm0-64c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12v8zm0-72v8c0 6.6-5.4 12-12 12H108c-6.6 0-12-5.4-12-12v-8c0-6.6 5.4-12 12-12h168c6.6 0 12 5.4 12 12zm96-114.1v6.1H256V0h6.1c6.4 0 12.5 2.5 17 7l97.9 98c4.5 4.5 7 10.6 7 16.9z"
                  fill="currentColor"
                  style={Object {}}
                />
              </svg>
            </span>
            Contact form
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
